lib/send_sms.rb

```diff
@@ -0,0 +1,18 @@
+# Download the helper library from https://www.twilio.com/docs/ruby/install
+require 'rubygems'
+require 'twilio-ruby'
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+account_sid = ENV['private']
```

Comment:
This code should be wrapped in the class that is responsible to send the text message, so probably inside the `Phone` class, in the method `receive` (see my comments above on that method)?